index.d.ts

```diff
@@ -0,0 +1,365 @@
+/// <reference types="node"/>
+import {ChildProcess} from 'child_process';
+import {Stream, Readable} from 'stream';
+
+export type StdIOOption =
+	| 'pipe'
+	| 'ipc'
+	| 'ignore'
+	| 'inherit'
+	| Stream
+	| number
+	| null
+	| undefined;
```

Comment:
Can you use the Node.js types instead?

Reply:
There is actually no type in node types that fits the type required here. `StdioOptions` fro node's `child_process` is not really a match here because of the convenience options `std{in,out,err}`, that split up this type so that we can't use it any more directly from node's types.